src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupService.scala

```diff
@@ -87,6 +87,10 @@ class ManagedGroupService(private val resourceService: ResourceService, private
     } yield ()
   }
 
+  def listGroups(userId: WorkbenchUserId): Future[Set[ResourceIdAndPolicyName]] = {
+    accessPolicyDAO.listAccessPolicies(ManagedGroupService.managedGroupTypeName, userId)
```

Comment:
if we add other policies they will show up here too... should we be proactive about dropping them?

Reply:
Yes, they will show up.  @breilly2 and I just talked about this a bit and I'm of the mind not to be proactive yet.  I think we're going to be addressing this issue as we try to tackle the "Request Access" GAWB-3335 problem.